packages/admin-frontend/pages/admin/edit/book.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+/**
+ * Part of GDL gdl-frontend.
+ * Copyright (C) 2018 GDL
+ *
+ * See LICENSE
+ */
+import * as React from 'react';
+import Typography from '@material-ui/core/Typography/Typography';
+import Container from '../../../components/Container';
+import EditBookForm from '../../../components/EditBook/EditBookForm';
+import EditTabBar from '../../../components/EditBook/EditTabBar';
+import { fetchBook } from '../../../lib/fetch';
+import Layout from '../../../components/Layout';
+import type { BookDetails, Context } from '../../../types';
+
+type State = {
+  selectedTab: number
+};
+
+export default class EditBookPage extends React.Component<
+  { book: ?BookDetails, chapterId: string },
+  State
+> {
+  static async getInitialProps({ query }: Context) {
+    if (!query.id) {
+      return {};
+    }
+
+    const bookRes = await fetchBook(query.id, query.lang);
+    const chapterId = query.chapterId;
+
+    let book;
+    if (bookRes.isOk) {
+      book = bookRes.data;
+    }
+
+    return { book, chapterId };
+  }
+
+  state = {
```

Comment:
It seems to me that the Tab component itself handles which tab is selected, so I guess the state here should be removed.